### ISSUE TITLE
"export from" does not get caught by this rule

### ISSUE THREAD

**Issue by kasper573** [kajirikajiri/eslint-plugin-mui-path-imports#3]
It's possible to trick this rule by doing thing:

```typescript
export { Delete } from "@mui/icons-material"; // Should not be allowed
```

**Comment by kajirikajiri** [comment]
I don't think that syntax is currently supported.

When I type the syntax you sent me into [this site](https://astexplorer.net/), I see that it is an ExportNamedDeclaration.
I believe that modifying [this file](https://github.com/kajirikajiri/eslint-plugin-mui-path-imports/blob/main/lib/rules/mui-path-imports.js) will take care of the problem.

**Comment by kasper573** [comment]
It is actually supported, but in typescript.

See https://www.typescriptlang.org/docs/handbook/modules.html#re-exports

maybe that is out of scope for this plugin?

**Comment by kajirikajiri** [comment]
>maybe that is out of scope for this plugin?

I agree.

I can't check the operation now, I'll try later.

**Comment by kajirikajiri** [comment]
Sorry, i have verified and decided not to support the proposed export syntax.

The reason is that I don't think it is in the scope of support for this package, which I imagine from the package name.

However, we have confirmed that using the export syntax described in the issue will result in a very slow hot-reload for Next.js.

Personally, I don't like this and am thinking of creating another package.

Below are the results of my test.

Without export syntax
event - compiled client and server successfully in 1422 ms (939 modules)
With export syntax
event - compiled client and server successfully in 18s (10830 modules)

Also, the following syntax does not seem to slow down the process, so I will try to create a package to modify it this way.

```typescript
export * as Delete from "@mui/icons-material/Delete";
```

**Comment by kajirikajiri** [comment]
I will close this time.
